docs/release-notes/cloud/cloud-2023-10.md

```diff
@@ -18,7 +18,7 @@ These release notes are for the Codacy Cloud updates during October 2023.
 -   You can now get two additional coverage status reports on GitHub using the new faster Coverage engine, showing whether your diff coverage and coverage variation are up to standards or not as configured on the quality gate rules for your repository. See [how to activate this setting](../../repositories-configure/integrations/github-integration.md#new-coverage-engine) for a repository. (ALA-593)
 
     !!! note
-        This setting will soon be active by default for all repositories.
+        This setting has been active by default since Dec. 1st, 2023. See [how to update your setup to use the new Coverage status reports](./cloud-2023-11-23-new-coverage-engine-status-checks.md).
```

Comment:
```suggestion
        This setting has been active by default since November 23rd, 2023. See [how to update your setup to use the new Coverage status reports](./cloud-2023-11-23-new-coverage-engine-status-checks.md).
```